main.go

```diff
@@ -173,14 +153,105 @@ func wrapMain() error {
 
 }
 
+func configFromStdin(vargs *GAE, workspace *string) error {
+	// https://godoc.org/github.com/drone/drone-plugin-go/plugin
+	workspaceInfo := plugin.Workspace{}
+	plugin.Param("workspace", &workspaceInfo)
+	plugin.Param("vargs", vargs)
+	// Note this hangs if no cli args or input on STDIN
+	plugin.MustParse()
```

Comment:
this just [panics if something gets messed up](https://github.com/drone/drone-plugin-go/blob/master/plugin/param.go#L126-L131), so no error to handle nicely